ruby/tests/api-test.cc

```diff
@@ -0,0 +1,172 @@
+#include <catch.hpp>
+#include <leatherman/ruby/api.hpp>
+
+using namespace std;
+using namespace leatherman::ruby;
+
+TEST_CASE("api::eval", "[ruby-api]") {
+    SECTION("can load api and evaluate ruby code") {
+        auto ruby = api::instance();
+        REQUIRE(ruby);
+        ruby->initialize();
+        REQUIRE(ruby->initialized());
+
+        REQUIRE(ruby->get_load_path().size() > 0);
+
+        REQUIRE(ruby->to_string(ruby->eval("'foo'")) == "foo");
+    }
+}
+
+TEST_CASE("api::is_*", "[ruby-api]") {
+    auto ruby = api::instance();
+    REQUIRE(ruby);
+    ruby->initialize();
+    REQUIRE(ruby->initialized());
+
+    SECTION("can correctly identify nil values") {
+        REQUIRE(ruby->is_nil(ruby->nil_value()));
+        REQUIRE_FALSE(ruby->is_nil(ruby->true_value()));
+    }
+
+    SECTION("can correctly identify true and false values") {
+        REQUIRE(ruby->is_true(ruby->true_value()));
+        REQUIRE_FALSE(ruby->is_true(ruby->false_value()));
+        REQUIRE(ruby->is_false(ruby->false_value()));
+        REQUIRE_FALSE(ruby->is_false(ruby->true_value()));
+    }
+
+    SECTION("can correctly identify strings") {
+        REQUIRE(ruby->is_string(ruby->utf8_value("'I'm a string'")));
+        REQUIRE_FALSE(ruby->is_string(1));
```

Comment:
This looks strange, I wouldn't expect `1` to be a valid `VALUE`. Maybe use nil/true/false, or eval to generate a number.